devops/debian/postinst

```diff
@@ -36,6 +36,11 @@ esac
 
 # Automatically added by dh_installsystemd/12.10ubuntu1
 if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
+	# MSFT change: binplace osconfig.toml for the azure-identity-service only if not present
+	# Binplacing a pre-existing install breaks the current identity
+	if [ ! -e /etc/aziot/identityd/config.d/osconfig.toml ]; then
+		cp /etc/osconfig/osconfig.toml /etc/aziot/identityd/config.d/
```

Comment:
>/etc/osconfig/osconfig.toml [](http://example.com/codeflow?start=5&length=27)

This is the problem.

Reply:
/etc/osconfig by definition only contains files that are consumed at run time by OSConfig. Also, README.md describes this osconfig.toml to be binplaced at /etc/aziot/identityd/config.d/. Does not say when, just that this is the destination.